src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,194 @@
+package duke;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+/**
+ Class to process the Tasklist of the chatbot
+ */
+public class TaskList {
+
```

Comment:
Since all methods has it as parameter, maybe ArrayList<Task> could have been an attribute of this class for the sake of OO